src/views/Accessibility/AccessibiltyRequest/Create/index.tsx

```diff
@@ -61,11 +61,18 @@ const Create = () => {
   };
 
   const systems = useMemo(() => {
-    return data?.systems?.edges || [];
+    const systemsObj: { [id: string]: any } = {};
```

Comment:
I have no idea what `any` should _really_ be.

Should it be `GetSystems_systems_edges`? Should we be importing it from the `queries/types/*` files? What about associated ESLint errors?

Reply:
I did an import from that type, too, and renamed the import to deal with the lint errors. eg: `import {Type_with_underscored as nameForMe} from 'queries/types...'